ebpf/symtab/elf.go

```diff
@@ -148,8 +159,16 @@ func (et *ElfTable) load() {
 }
 
 func (et *ElfTable) createSymbolTable(me *elf2.MMapedElfFile) (SymbolNameResolver, error) {
-	symTable, symErr := me.NewSymbolTable()
 	goTable, goErr := me.NewGoTable()
+	if !et.options.SymbolOptions.GoTableFallback && goErr == nil {
+		return goTable, nil
+	}
+	symbolOptions := elf2.SymbolsOptions{}
+	if goErr == nil && goTable.Index.Entry.Length() > 0 {
+		symbolOptions.FilterTo = goTable.Index.Entry.Get(0)
+		symbolOptions.FilterTo = goTable.Index.End
```

Comment:
There must be a typo:
```go
symbolOptions.FilterFrom = goTable.Index.Entry.Get(0)
```

Reply:
yes!
I guess my test is broken as well :(

Reply:
actually I dont know how to easily catch it with a test. the test works because it filters `[0;last_go_symbol]`
And in order to catch this typo I need to have a symbol in `.sym` in range `[0;first_go_symbols]` which I dont know how to do without messing with a linker.

I will just fix the typo and not improve test

Thanks!